[2024-01-01T10:43:13.253+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: snow_dag.is_snow_api_ready manual__2024-01-01T10:43:08.544267+00:00 [queued]>
[2024-01-01T10:43:13.264+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: snow_dag.is_snow_api_ready manual__2024-01-01T10:43:08.544267+00:00 [queued]>
[2024-01-01T10:43:13.264+0000] {taskinstance.py:2171} INFO - Starting attempt 1 of 3
[2024-01-01T10:43:13.359+0000] {taskinstance.py:2192} INFO - Executing <Task(HttpSensor): is_snow_api_ready> on 2024-01-01 10:43:08.544267+00:00
[2024-01-01T10:43:13.362+0000] {standard_task_runner.py:60} INFO - Started process 468 to run task
[2024-01-01T10:43:13.365+0000] {standard_task_runner.py:87} INFO - Running: ['airflow', 'tasks', 'run', 'snow_dag', 'is_snow_api_ready', 'manual__2024-01-01T10:43:08.544267+00:00', '--job-id', '11', '--raw', '--subdir', 'DAGS_FOLDER/servicenow_dag.py', '--cfg-path', '/tmp/tmp3h2t7iu6']
[2024-01-01T10:43:13.369+0000] {standard_task_runner.py:88} INFO - Job 11: Subtask is_snow_api_ready
[2024-01-01T10:43:13.492+0000] {task_command.py:423} INFO - Running <TaskInstance: snow_dag.is_snow_api_ready manual__2024-01-01T10:43:08.544267+00:00 [running]> on host cdceb6054911
[2024-01-01T10:43:13.860+0000] {taskinstance.py:2481} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='prady1900' AIRFLOW_CTX_DAG_ID='snow_dag' AIRFLOW_CTX_TASK_ID='is_snow_api_ready' AIRFLOW_CTX_EXECUTION_DATE='2024-01-01T10:43:08.544267+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-01-01T10:43:08.544267+00:00'
[2024-01-01T10:43:13.861+0000] {http.py:130} INFO - Poking: api/now/table/incident
[2024-01-01T10:43:13.888+0000] {base.py:83} INFO - Using connection ID 'snow_api_id' for task execution.
[2024-01-01T10:43:17.428+0000] {base.py:293} INFO - Success criteria met. Exiting.
[2024-01-01T10:43:17.439+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=snow_dag, task_id=is_snow_api_ready, execution_date=20240101T104308, start_date=20240101T104313, end_date=20240101T104317
[2024-01-01T10:43:17.557+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-01-01T10:43:17.683+0000] {taskinstance.py:3281} INFO - 1 downstream tasks scheduled from follow-on schedule check
